cytoscape-panzoom.js

```diff
@@ -513,10 +577,41 @@ SOFTWARE.
               return false;
             });
 
+            $button.bind("touchstart", function(e) {
+              e.preventDefault();
+              e.stopPropagation();
+
+              var doZoom = function() {
+                let zoom = cyRef.zoom();
+                let lvl = cyRef.zoom() * factor;
+
+                if (lvl < options.minZoom) {
+                  lvl = options.minZoom;
+                }
+                if (lvl > options.maxZoom) {
+                  lvl = options.maxZoom;
+                }
+                if ((lvl == options.maxZoom && zoom == options.maxZoom) ||
```

Comment:
Could knock this down to 3 clauses by checking `lvl === zoom`, but again probably not critical.